### ISSUE TITLE
[package] Regression on Windows 2008R2

### ISSUE THREAD

**Issue by jonathanmorley** [inspec/inspec#998]
### Description

```
describe package('Datadog Agent') do
  it { should be_installed }
end
```

should pass tests when inspec is run on a machine with Datadog installed.
### InSpec and Platform Version

inspec 0.26.0 passes
inspec 0.32.0 fails

Both tests run against Windows 2008R2 64bit
### Possible Solutions

Revert https://github.com/chef/inspec/issues/86

**Comment by chris-rock** [comment]
@jonathanmorley Is this not working with the datadog package only or in general?

**Comment by chris-rock** [comment]
@jonathanmorley Is this happening on Windows 2012 too?

**Comment by jonathanmorley** [comment]
Unknown on both counts, I haverolled back my inspec install to 0.29.0 to work around this issue for now. If I get some time I will try to test other scenarios

**Comment by chris-rock** [comment]
We've done integration testing to ensure it works exactly the same as the old one. See https://github.com/chef/inspec/blob/master/test/integration/default/package_spec.rb#L34-L58

Lets figure out if this is restricted to Windows 2008 or also not working on Windows 2012

**Comment by chris-rock** [comment]
@jonathanmorley I'll scheduled further investigation for next week. Let me know if you have further insights until then.

**Comment by chris-rock** [comment]
May be related to #1100 and https://github.com/chef/train/pull/151
@jonathanmorley Are you using 64bit?

**Comment by chris-rock** [comment]
Just saw you are. Our bit detection was returning the wrong values. Will be fixes with the next version of train

**Comment by chris-rock** [comment]
This has been solved with train 0.20.0 https://github.com/chef/train/releases/tag/v0.20.0 via https://github.com/chef/train/pull/151. Please reopen if the issue persists
